flare/struc.py

```diff
@@ -127,9 +130,49 @@ def get_cell_dot(self):
 
         return cell_dot
 
+    def get_max_cutoff(self) -> float:
+        """Compute the maximum cutoff compatible with a 3x3x3 supercell of the
```

Comment:
This function is nice! Because it takes no arguments and is only used once during the setup of the function, it could be safely moved to a `util` file, and renamed more descriptively via `get_max_cutoff_for_cell` with a more thorough explanation of why it exists (since the util files will be much less frequently scrolled through than `struc.py`).

Reply:
P.S. It would also just take `cell` as an argument.

Or, if you have strong feelings that ripping it away from the `Structure` class doesn't make sense, it could also be moved away from a method of `Structure` and as a function defined at the bottom of the file.

Reply:
Method has been moved to util.py